api/everest-server.gen.go

```diff
@@ -5747,6 +5882,21 @@ type ServerInterface interface {
 	// Update database engine
 	// (PUT /namespaces/{namespace}/database-engines/{name})
 	UpdateDatabaseEngine(ctx echo.Context, namespace string, name string) error
+	// List Split-Horizon DNS Config instances.
+	// (GET /namespaces/{namespace}/engine-features/split-horizon-dns-config)
+	ListSplitHorizonDNSConfigs(ctx echo.Context, namespace string) error
+	// Create Split-Horizon DNS Config for PSMDB.
+	// (POST /namespaces/{namespace}/engine-features/split-horizon-dns-config)
+	CreateSplitHorizonDNSConfig(ctx echo.Context, namespace string) error
+	// Delete Split-Horizon DNS Config instance.
+	// (DELETE /namespaces/{namespace}/engine-features/split-horizon-dns-config/{name})
+	DeleteSplitHorizonDNSConfig(ctx echo.Context, namespace string, name string) error
+	// Get Split-Horizon DNS Config instance.
+	// (GET /namespaces/{namespace}/engine-features/split-horizon-dns-config/{name})
+	GetSplitHorizonDNSConfig(ctx echo.Context, namespace string, name string) error
+	// Update Split-Horizon DNS Config instance.
+	// (PATCH /namespaces/{namespace}/engine-features/split-horizon-dns-config/{name})
+	UpdateSplitHorizonDNSConfig(ctx echo.Context, namespace string, name string) error
```

Comment:
Some points to consider renaming to plural (`split-horizon-dns-configs`)
- Consistency: in all other endpoints we have plural naming for resource in API (`backup-storages`, `backups`, `database-clusters` etc)
- Using plural nouns to name collection URIs is aligned with REST API best practices
- In RBAC the resource is named `split-horizon-dns-configs`
- `GET /namespaces/{namespace}/engine-features/split-horizon-dns-config` looks confusing and it's hard to say it's going to return a list of configs.